### PR TITLE
Manual pages for modpack installer tools

### PR DESCRIPTION
**Pull request by jwrober**
Part of #1947

### CODE REVIEW

File: docs/Manuals/Program/freeciv21-modpack-qt.rst [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  :end-line: 29
+
+This is a program to read and install a modpack URL from the command line. When given no command line
+parameters the Qt base GUI will load and allow for modpack installation. For a pure command line modpack
+installer, see freeciv21-modpack(6).
+
+OPTIONS
+=======
+
+The following options are accepted on the command line of the manual generator. They may not be combined as
```
Comment (daavko) [inline docs/Manuals/Program/freeciv21-modpack-qt.rst:26]: I'm guessing "manual generator" in this sentence should be "modpack installer"?

File: docs/Manuals/Program/freeciv21-modpack.rst [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  :start-line: 17
+  :end-line: 29
+
+This is a program to read and install a modpack URL from the command line. If you are looking for a GUI
+based modpack installer, refer to freeciv21-modpack-qt(6).
+
+OPTIONS
+=======
+
+The following options are accepted on the command line of the manual generator. They may not be combined as
```
Comment (daavko) [inline docs/Manuals/Program/freeciv21-modpack.rst:25]: Same here, "manual generator" should be "modpack installer" or something like that?

**Review by daavko — COMMENTED** [review]
Two small things, other than that LGTM

File: docs/Manuals/Program/freeciv21-modpack-qt.rst (hunk 1 above)
Comment (jwrober) [reply docs/Manuals/Program/freeciv21-modpack-qt.rst:26]: Oh good catch. I copied the -manual.rst page.

**Pull request merged**